openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -100,6 +100,11 @@ def can_show_upgrade_sock(self):
         can_show = can_show_verified_upgrade(self.effective_user, self.enrollment_object)
         return can_show
 
+    @property
+    def license(self):
+        course = get_course_by_id(self.course_key)
+        return course.license
```

Comment:
I'd hold off with the approval until we verify whether we'll be handling the undefined license on the backend by returning the default one.

Reply:
Is this comment resolved?

Reply:
@davidjoy, yep, I just didn't have permissions to mark it as resolved.